src/ct/codetracerconf.nim

```diff
@@ -173,44 +173,49 @@ type
        abbr: "i",
        desc: "explicit flag for interactively choosing a trace"
       .}: Option[bool]
-    # of build:
-    #   buildProgramPath* {.
-    #     argument
-    #     desc: "path to program source code"
-    #   .} : string
-    #   buildOutputPath* {.
-    #     argument
-    #     defaultValue: ""
-    #     desc: "Output path: by default the program path without the extension"
-    #   .} : string
+    of build:
+      buildProgramPath* {.
+        argument
+        desc: "path to program source code"
+      .} : string
+      buildOutputPath* {.
+        argument
+        desc: "Output path: by default the program path without the extension"
+      .} : string
     of record:
```

Comment:
it would be great if this works now

IIRC not sure this had custom handling before, because we needed some special parsing logic, or maybe because of `restOfArgs`: I haven't written down why wasn't the normal confutils api used though;